firmware/lib/Axis/src/Axis.h

```diff
@@ -0,0 +1,63 @@
+#ifndef AXIS_H
+#define AXIS_H
+
+#include "Debug.h"
+#include "Kinematics.h"
```

Comment:
Only include in the header file, what's needed by the header file. If it's needed by the cpp file you can include it directly in the cpp file. So I think both of these includes can be moved to Axis.cpp